src/display/canvas.js

```diff
@@ -1313,11 +1313,20 @@ class CanvasGraphics {
       0,
     ]);
     maskToCanvas = Util.transform(maskToCanvas, [1, 0, 0, 1, 0, -height]);
-    const cord1 = Util.applyTransform([0, 0], maskToCanvas);
-    const cord2 = Util.applyTransform([width, height], maskToCanvas);
-    const rect = Util.normalizeRect([cord1[0], cord1[1], cord2[0], cord2[1]]);
-    const drawnWidth = Math.round(rect[2] - rect[0]) || 1;
-    const drawnHeight = Math.round(rect[3] - rect[1]) || 1;
+    const cords = [
+      [0, 0],
+      [width, height],
+      [0, height],
+      [width, 0],
+    ].map(cord => Util.applyTransform(cord, maskToCanvas));
+    const xList = cords.map(cord => cord[0]);
+    const yList = cords.map(cord => cord[1]);
+    const minX = Math.min(...xList);
+    const minY = Math.min(...yList);
+    const maxX = Math.max(...xList);
+    const maxY = Math.max(...yList);
+    const drawnWidth = Math.round(maxX - minX);
+    const drawnHeight = Math.round(maxY - minY);
```

Comment:
The old code ensured that these are non-zero, why was that removed?
```suggestion
    const drawnWidth = Math.round(maxX - minX) || 1;
    const drawnHeight = Math.round(maxY - minY) || 1;
```